packages/react-menu/src/MenuList/MenuList.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { useMenuList } from './useMenuList';
+import { MenuListProps } from './MenuList.types';
+import { renderMenuList } from './renderMenuList';
+
+/**
+ * Define a styled MenuList, using the `useMenuList` hook.
+ * {@docCategory MenuList}
+ */
+export const MenuList = React.forwardRef<HTMLElement, MenuListProps>((props, ref) => {
+  const state = useMenuList(props, ref);
+
```

Comment:
IMO, we should have some styles for `MenuList`.

May be we should define `font` at least:

![image](https://user-images.githubusercontent.com/14183168/107210213-1d5d4f80-6a04-11eb-9809-81c23b34fee9.png)

Reply:
I disagree here, consumers can have some styling themselves, we will use this in a popup component at some point which should be styled

If you read the spec, the purpose of the `MenuList` is to render on a consumers own surface, whatever that might be

Reply:
The styles I used in the stories for the container, is essentially design spec for the popup container